core/src/main/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandlerBase.java

```diff
@@ -211,7 +201,13 @@ public void onThrottleReady(boolean wasDelayed) {
           System.nanoTime() - startTimeNanos,
           TimeUnit.NANOSECONDS);
     }
-    sendRequest(null, 0, 0, true);
+    Queue<Node> queryPlan =
+        this.statement.getNode() != null
```

Comment:
it is used from one thread? If not we will need to use AtomicReference or something similar